examples/nlp/question_answering/conf/qa_conf.yaml

```diff
@@ -0,0 +1,157 @@
+  library: huggingface # [huggingface, megatron]. Used by S2SQAModel and GPTQAModel
+  save_model: False # save validation model checkpoints
+
+  tokens_to_generate: 32 # used by S2SQAModel and GPTQAModel to limit number of generated tokens
+
+  dataset:
+    version_2_with_negative: true # if true, dataset contains some questions that do not have an answer
+    doc_stride: 128 # stride for splitting long documents into chunks
+    max_query_length: 64
+    max_seq_length: 512 # max sequence length for input to the model
+    max_answer_length: 30 # max ground truth answer length
+    use_cache: false
+    do_lower_case: true
+
+    # if true, context spans/chunks that do not contain answer are treated as unanswerable,
+    #   useful for extractive datasets like SQuAD
+    # if false, all context spans/chunks are treated as relevant for answering given query,
+    #   useful for generative datasets where answer is not necessarily in the context
+    # used by S2SQAModel and GPTQAModel   
+    check_if_answer_in_context: true
+
+    # if all, keep all doc spans
+    # if only_positive, keep doc spans containing answer only
+    # if limited_negative, keep 10 doc spans closest to answer per question
+    # used by BERTQAModel
+    keep_doc_spans: all # [all, only_positive, limited_negative]
+
+    null_score_diff_threshold: 0.0 # If null_score - best_non_null is greater than the threshold predict null.
+    n_best_size: 20
+
+    num_workers: 1
+    pin_memory: false
+    drop_last: false
+
+  train_ds:
+    file: null # .json file
+    batch_size: 24 # per GPU
+    shuffle: true
+    num_samples: -1
+
+    # default values for the following params are retrieved from dataset config section, but you may override them
+    num_workers: ${model.dataset.num_workers}
+    drop_last: ${model.dataset.drop_last}
+    pin_memory: ${model.dataset.pin_memory}
+
+  validation_ds:
+    file: null # .json file
+    batch_size: 24 # per GPU
+    shuffle: false
+    num_samples: -1
+
+    # default values for the following params are retrieved from dataset config section, but you may override them
+    num_workers: ${model.dataset.num_workers}
+    drop_last: ${model.dataset.drop_last}
+    pin_memory: ${model.dataset.pin_memory}
+
+  test_ds:
+    file: null # .json file
+    batch_size: 24 # per GPU
+    shuffle: false
+    num_samples: -1
+    
+    # default values for the following params are retrieved from dataset config section, but you may override them
+    num_workers: ${model.dataset.num_workers}
+    drop_last: ${model.dataset.drop_last}
+    pin_memory: ${model.dataset.pin_memory}
+
+  language_model:
+    pretrained_model_name: bert-base-uncased # main config to select model (between bert, gpt2, t5/bart based models)
+    lm_checkpoint: null
+    config_file: null # json file, precedence over config
+    config: null
+
+  token_classifier: # used only by BERTQAModel for defining the extractive QA head
```

Comment:
Be careful to make sure that the default args are kept generally consistent i.e. there might be some people try to call the bert QA model based before this change was made so we would want to make it easy for them to do so